feature-libs/product-configurator/rulebased/core/state/actions/configurator.action.ts

```diff
@@ -185,6 +185,7 @@ export class UpdateConfigurationFinalizeSuccess extends StateUtils.EntitySuccess
   }
 }
 
+// eslint-disable-next-line @nx/workspace/no-ngrx-fail-action-without-error-action-implementation
```

Comment:
Let's add explanation why it's ignored here after `... -- ...` + leave JIRA ticket number here, for reference, to sort it out in the near future. + let's contact Team Tiger to take a look at it

Reply:
Sure, here's the follow-up ticket: https://jira.tools.sap/browse/CXSPA-7901